src/Http/Controllers/API/CollectionTreeController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Statamic\Http\Controllers\API;
+
+use Statamic\Exceptions\NotFoundHttpException;
+use Statamic\Facades\Site;
+use Statamic\Http\Resources\API\TreeResource;
+
+class CollectionTreeController extends ApiController
+{
+    public function show($collection)
+    {
+        $site = request('site', Site::default()->handle());
+
+        $tree = $collection->structure()->in($site);
```

Comment:
There's an error here if you visit the endpoint of a non-structured collection, like blog. Gotta work a `hasStructure()` check in here.